Add Monster Hunter 3 Ultimate

brunomenezes66 opened the pull request:
Adds Monster Hunter 3 Ultimate.

*Automatically generated by PR builder*

TheKoopaKingdom commented:
Please add a testcase to your game.dat file.

brunomenezes66 commented:
@TheKoopaKingdom Please explain yourself a little bit better since I'm not familiarized with the extension or the emulator at all.

TheKoopaKingdom commented:
Don't worry about it, it's actually fine the way it is (But if you do want to learn more about the format, the documentation is [here](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md#code)).

TheKoopaKingdom commented on monster-hunter-3-ultimate/game.dat:
```diff
@@ -0,0 +1,14 @@
+title = "Monster Hunter 3 Ultimate"
+description = "RPG hunting game. Publisher: Capcom"
```
This description isn't that good, you can get a longer one from [Wikipedia](https://en.wikipedia.org/wiki/List_of_Nintendo_3DS_games).

brunomenezes66 commented:
@TheKoopaKingdom Just updated the game.dat, I hope It`s better now, but can`t update de boxart (or at least I don`t know how) without installing the GitHub app, which I can`t do right now.

Schplee commented:
@brunomenezes66 Please fix the remaining issues, otherwise we will close the PR.

brunomenezes66 commented:
@Schplee @TheKoopaKingdom I fixed all the requested problems:

- The boxart has been updated even tho it doesn't show the 3ds logo here in the GitHub page (it does in the GitHub desktop version)
- I updated the game.dat and the game description.
- I also changed the folder name in order to have the proper capitalization.
I'm still waiting for the wiki to be updated or at least for somebody to give me some feedback in order to fix the problems/issues section.

Schplee commented:
@brunomenezes66 The boxart isn't updated, and there's no commit indicating you've updated the OR to reflect any changes in the Bogart.

Also, the wiki shouldn't have a `Problems` section, as any issues should be linked via github in the metadata. Please remove it. Also, the description is inaccurate, as the game runs but doesn't display menu graphics due to missing geometry sharers. Please revise the wiki article to be verbose as dictated by our contributions guide, and include the explanation I provided. Thank you!

brunomenezes66 commented:
@Schplee I just linked the github issues in the game.dat but I still don´t know where is the problems section that you mention.
Also, in my first commit I actually changed the boxart, It just doesn´t show there. I just tried to change the boxart again through GitHub Desktop but it doesn´t even show the commit at all.
Don´t really know how to fix that so you can close and let somebody else do it from scratch, maybe somebody is luckier than my :(

brunomenezes66 commented:
@Schplee

Schplee commented:
@brunomenezes66 I see. The rest of your PR is good, so I will merge it and upload the fixed boxart.

However, before I merge it, please modify the Wiki article: https://github.com/citra-emu/citra-games-wiki/wiki/Monster-Hunter-3-Ultimate

This is what I was referring to in my previous message.

TheKoopaKingdom commented:
>I also changed the folder name in order to have the proper capitalization.

It looks like the folder name is still all lower-case.

Schplee commented:
@TheKoopaKingdom Folder names are supposed to be lower-case.
See #73

brunomenezes66 commented:
@Schplee Already fixed the wiki as requested. Please close this PR, is getting way too big for something so simple. Also if you want the updated boxart with the 3DS logo I still have it so you don't have to be wasting time with that.

Schplee commented:
@brunomenezes66 Thank you, but I already have the boxart ready. This PR can be merged now.
